managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -586,6 +588,47 @@ public void asyncReadEntries(int numberOfEntriesToRead, long maxSizeBytes, ReadE
         ledger.asyncReadEntries(op);
     }
 
+    public CompletableFuture<Entry> readSpecifyPositionByPosition(PositionImpl position) {
```

Comment:
We already have a method `asyncReadEntry` in the ManagedLedgerImpl, it's better to use the current implementation directly.